tests/src/Libs/WP_Application_Test.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Tests\Unit;
```

Comment:
We need to use the namespaces like the updated develop (you can see the previous test file in `unit` folder)